pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -82,7 +82,6 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 	accesscontrol.AddManagedPermissionsMigration(mg, accesscontrol.ManagedPermissionsMigrationID)
 	accesscontrol.AddManagedFolderAlertActionsMigration(mg)
 	accesscontrol.AddActionNameMigrator(mg)
-	addPlaylistUIDMigration(mg)
```

Comment:
We never modify past migrations - the migration ID (which is the name) is stored in the database, in order, and we aim to preserve that record. Only additions, no removals.